.gitignore

```diff
@@ -12,3 +12,4 @@ pom.xml.asc
 *.swo
 /docs-tmp/
 /resources/configuration-local.edn
+/figwheel_server.log
```

Comment:
<3 figwheel

Also kinda disappointed it's a cljs tool and not a dense fruit dessert